src/types.ts

```diff
@@ -120,7 +121,7 @@ export type AutocompleteSearchSubmit = {
  * @example if (isAutocompleteSelectSubmit(event)) { ... } // `query` is available
  */
 export const isAutocompleteSelectSubmit = (
-  event: AutocompleteSubmitEvent
+  event: AutocompleteSubmitEvent,
```

Comment:
the linter is failing here

Reply:
Good catch, looks like the right configuration was `"trailingComma": "es5"` instead of "all"